barcodes_generator_package/models/product_packaging.py

```diff
@@ -0,0 +1,12 @@
+# Copyright 2016 LasLabs Inc.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import fields, models
+
+
+class ProductPackaging(models.Model):
+    _name = 'product.packaging'
+    _description = 'Product Packaging'
+    _inherit = ['product.packaging', 'barcode.generate.mixin']
+
+    barcode = fields.Char()
```

Comment:
@misern2 Did you see my PR that reimplement former barcode field (product_packaging_barcode) ? #153